pkg/pillar/cmd/zedagent/zedagent.go

```diff
@@ -98,6 +98,8 @@ type zedagentContext struct {
 	rebootCmd                 bool
 	rebootCmdDeferred         bool
 	deviceReboot              bool
+	devInfoTriggerFail        bool
+	devInfoTaskReady          bool
```

Comment:
Can you add comments on what each of these used for?

devInfoTaskReady --> This indicates if a publish is currently in progress - right?
   - So, this must be set just before Publish is called - And reset after Publish is Done??

devInfoTriggerFail
   - This is really an indication if the last publish failed - right? And if it failed, you want to retry the publish after some trigger.. right?
   - So, should we name it as devInfoTaskSuccess? Success if True / Failed if False?

Reply:
sure. will add appropriate comments.